app/helpers/katello/hosts_and_hostgroups_helper.rb

```diff
@@ -77,20 +61,19 @@ def kickstart_repository_id(host, options = {})
     end
 
     def fetch_lifecycle_environment(host, options = {})
+      return host.lifecycle_environment if host.lifecycle_environment.present?
       selected_host_group = options.fetch(:selected_host_group, nil)
       return lifecycle_environment(selected_host_group) if selected_host_group.present?
```

Comment:
looks like you deleted the lifecycle_environment method above:

undefined method `lifecycle_environment' for #<#<Class:0x0000000bb7e3e0>:0x0000000bb8ed58>

Reply:
this seems to happen if you create a host, select the host group, but don't select a CV or LCE.  Then go to the host edit page.

It does seem to solve the original issue thou gh.